src/server/raft_client.rs

```diff
@@ -68,7 +68,8 @@ impl Conn {
         let client = TikvClient::new(channel);
         let (tx, rx) = mpsc::unbounded();
         let (tx_close, rx_close) = oneshot::channel();
-        let (sink, _) = client.raft().unwrap();
+        let opt = CallOption::default().timeout(cfg.grpc_raft_timeout.0);
+        let (sink, _) = client.raft_opt(opt).unwrap();
```

Comment:
Have you tested it? Last time I check it will timeout after `grpc_raft_time` even if there are communications on the connection. I think the proper way to solve it is using the keep alive configuration. /cc @disksing

Reply:
Yep, test shows that this will keep timeout all the time, but is it expected or am I using it wrong?

Reply:
It's expected. The timeout here means how long the call should last for most.